lib/src/Uncertainty/Algorithm/MetaModel/Kriging/GaussianProcessRegressionResult.cxx

```diff
@@ -39,7 +39,7 @@ GaussianProcessRegressionResult::GaussianProcessRegressionResult()
 /* Constructor with parameters  */
 GaussianProcessRegressionResult::GaussianProcessRegressionResult(const GaussianProcessFitterResult & result,
     const Sample & covarianceCoefficients)
-  : MetaModelResult(result.getInputSample(), result.getOutputSample(), result.getMetaModel(), result.getResiduals(), result.getRelativeErrors())
+  : MetaModelResult(result.getInputSample(), result.getOutputSample(), result.getMetaModel())
```

Comment:
yyou can drop this commit; I made a simpler change in #2930 to use tMetaModelResult(result) instead which is simpler

Reply:
ok